modules/playarte.crawler.class.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+// Require node_modules dependencies
+let path = require('path');
+let cheerio = require('cheerio');
+let moment = require('moment');
+let _ = require('lodash');
+
+// Require other classes, files or configs
+let MainCrawler = require(path.join(__dirname, '../modules', 'main.crawler.class'));
+
+module.exports = class PlayarteCrawler extends MainCrawler {
+    getPage(url) {
+        return new Promise((resolve, reject) => {
+            super.getStaticPage(url)
+                .then(function($){
```

Comment:
You need other method to collect and process the information.

See: 

- https://github.com/fernandoguedes/gollum/blob/master/modules/cinemark.crawler.class.js#L15-L19

- https://github.com/fernandoguedes/gollum/blob/master/modules/cinemark.crawler.class.js#L23-L79